modules/custom-status/custom-status.php

```diff
@@ -252,6 +248,7 @@ public function action_admin_enqueue_scripts() {
 
 			wp_localize_script( 'vip-workflow-custom-status-configure', 'VW_CUSTOM_STATUS_CONFIGURE', [
 				'custom_statuses'    => $this->get_custom_statuses(),
+				'editorial_metadatas' => EditorialMetadata::get_editorial_metadata_terms(),
```

Comment:
There's taxonomy conflicts that happen due to the different inits calling `get_terms`/`get_term`. As a result, this is way safer to do so the client can handle the mapping. We could do some mapping but I didn't want more conflicts happening here to.

Perhaps after custom status is migrated, we can revisit this. I didn't feel the refactor would fix this but I could be wrong.